README.md

```diff
@@ -10,29 +12,33 @@ The app is a React/Redux app which lets you draw on a `<canvas>` element with di
-* Save: a button that will take what you've drawn in the canvas and open it as an image in another tab so you can save it.
+- [ ] Colorpicker: a component that lets you pick a color to draw. The built in HTML color input will be fine here.
+- [ ] Image stamp: a place for you to upload an image and pop it in the drawing area like so. The image stamp component should give you a preview of the image uploaded. It should function like this: ![Image Stamp Tool](/public/img/stamp.gif?raw=true "Image Stamp Tool")
+- [ ] Eraser: a tool to erase what you've drawn. Hint: there is a particular color that makes for a great eraser.
+- [ ] Reset: a button that wipes the entire canvas clean so you can make a new drawing.
+- [ ] Save: a button that will take what you've drawn in the canvas and open it as an image in another tab so you can save it.
 
-Also you'll need to style the sidebar so it looks as close to this image as possible. **Clean, maintainable, and organized (S)CSS is an important part of your submission!**
+You'll also need to style the sidebar so it looks as close to this image as possible. **Clean, maintainable, and organized (S)CSS is an important part of your submission!**
 ![Design](/public/img/design.png?raw=true "Design")
 
 It only needs to work in the latest Chrome & Firefox.
 
 ## How to submit
-Send all your files in one email to internchallenge@socialtables.com with the subject line, Social Tables Venue Mapper Intern Challenge.
-
-You must include the following files in your email: YourInitials-readme.{md, txt}, YourInitials-code.{tar.gz, zip}, YourInitials-resume.pdf (where YourInitials are your initials).
+Send all your files in one email to [internchallenge@socialtables.com](mailto:internchallenge@socialtables.com) with the subject line: `Social Tables Venue Mapper Intern Challenge`.
 
 These files will contain:
 
-* The YourInitials-readme.{md, txt} contains instructions for running your code and any notes you wish to include.
-* The YourInitials-code.{tar.gz, zip} contains your code. You can optionally also submit a link to your code in the readme file.
-* A file named YourInitials-resume.pdf containing your resume.
-
-Submitting your node-modules folder isn't necessary, for example, because we can get it by running npm install; however, you should at least submit any files you found necessary to change and enough support and explanation so we can reproduce your results. In any case, include all the information in YourInitials-readme.{md, txt}.
+* `YourInitials-readme.{md, txt}` containing:
+    * Instructions for running your code
+    * Any notes you wish to include
+    * **Brief answers to the following questions**:
+        1. What aspect of your submission are you most proud of, and why?
+        1. What would you work on if you had more time?  
+        1. Of all the resources (blog posts, tutorials, videos, etc.) you referenced, what was most helpful for you?
```

Comment:
@jschultz21 Good call! I think I'll just add "mentors". If we have meetup groups on the list, it might give the impression we're explicitly down for groups working on this together? What do you think?